pushMagic.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Personal development files should not be added to the public repo. These should be removed. You can ignore them easily by configuring a global .gitignore that applies to all your projects. https://stackoverflow.com/questions/7335420/global-git-ignore